src/core/declarativewebcontainer.cpp

```diff
@@ -962,3 +973,10 @@ void DeclarativeWebContainer::drawUnderlay()
         funcs->glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
     }
 }
+
+void DeclarativeWebContainer::handleContentOrientationChanged(Qt::ScreenOrientation orientation)
+{
+    if (orientation == pendingWebContentOrientation()) {
```

Comment:
I am not sure what is the motivation for this method. I see below code in ``` void QMozWindowPrivate::timerEvent(QTimerEvent *event)``` method:

```
            if (mOrientation != mPendingOrientation) {
                 mWindow->SetContentOrientation(QtToMozillaRotation(mPendingOrientation));
                 mOrientation = mPendingOrientation;
             } else {
                 q.orientationChangeFiltered(mOrientation);
             }
```

It means this event should be ignored when pending orientation is same as current orientation.